app/models.py

```diff
@@ -1,3 +1,81 @@
+from zoneinfo import available_timezones
 from django.db import models
 
-# Create your models here.
+import uuid
+
+class Grant(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    name = models.CharField(max_length=255, null=False)
+    
+    def __str__(self):
+        return f" {self.name} Grant {self.id}"
```

Comment:
NIT: In this case, `Grant(id=1,name="Grant pre vývoj akademických experimentov)` would print out as `' Grant pre vývoj akademických experimentov Grant 1'`.

What about the following?
```suggestion
        return f"Grant(id={self.id},name={self.name})"
```